### PR TITLE
Create TrashToken

### PR DESCRIPTION
**Pull request by webmaster128**
an ERC20 without name, symbol, decimals

### CODE REVIEW

File: test/TrashToken.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
+const chai = require("chai");
+const chaiAsPromised = require("chai-as-promised");
+const { expect } = chai;
+chai.use(chaiAsPromised);
+const BN = require("bn.js");
+chai.use(require("chai-bn")(BN));
+const { expectEvent } = require("openzeppelin-test-helpers");
```
Comment (willclarktech) [inline test/TrashToken.js]: How do you feel about putting some of this setup into a separate file?

File: test/TrashToken.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  describe("transfer()", () => {
+    const tokenOwner = accounts[0];
+
+    before(async () => {
+      await testContract.mint(tokenOwner, 4, { from: minter });
+    });
+
+    it("works", async () => {
+      const recipient = makeRandomAddress();
+      const { tx } = await testContract.transfer(recipient, 4);
```
Comment (willclarktech) [inline test/TrashToken.js]: I wouldn't mind making the sender explicit by convention in these cases (even though it's not necessary for `accounts[0]`).

**Review by willclarktech — COMMENTED** [review]
Looks good. What's the purpose though?

File: test/TrashToken.js (hunk 2 above)
Comment (webmaster128) [reply test/TrashToken.js]: Yeah I agree that all transaction calls (signed, state changes) should have an explicit sender. I did that for most of them already. Will update this case.

**Comment by webmaster128** [comment]
> Looks good. What's the purpose though?

This is a Testing token that is minimal, i.e. does not have the optional fields name, symbol, decimals. We need it in IOV-Core to test properly handling of those kind of tokens.

File: test/TrashToken.js (hunk 1 above)
Comment (webmaster128) [reply test/TrashToken.js]: I'd be happy to see how much better it can get. I don't think much because we need some of those symbold available locally.

Comment (willclarktech) [reply test/TrashToken.js]: Two options would be:
1. Store things like `expectEvent` on the global object; or
2. Export them from the setup file and destructure what we need in a single line from the test file. (Preferred)

**Pull request merged**